Truck.js

```diff
@@ -1,6 +1,39 @@
 var Car = require('./Car.js');
 
 //Implement your Truck here
-function Truck() {};
+function Truck(make, model, year, color, capacity) {
+	Car.call(this);
```

Comment:
Similarly, here I would include the parameters inherited from car from within the ```Car.call()``` function.
```
function Truck(make, model, year, capacity) {
	Car.call(this, make, model, null, "Dystopian Blue");
```